.env.template

```diff
@@ -121,7 +121,36 @@ MODELS=`[
           "prompt": "How do I make a delicious lemon cheesecake?"
         }
       ]
-    }
+    },
+      {
+    "name": "openchat/openchat_3.5",
+    "displayName": "openchat/openchat_3.5",
+    "description": "OpenChat 3.5 is the #1 model on MT-Bench, with only 7B parameters.",
+    "websiteUrl": "https://huggingface.co/openchat/openchat_3.5",
+    "preprompt": "",
+    "chatPromptTemplate" : "<s>{{#each messages}}{{#ifUser}}GPT4 Correct User: {{#if @first}}{{#if @root.preprompt}}{{@root.preprompt}}\n{{/if}}{{/if}}{{content}}<|end_of_turn|>GPT4 Correct Assistant:{{/ifUser}}{{#ifAssistant}}{{content}}<|end_of_turn|>{{/ifAssistant}}{{/each}}",
```

Comment:
I don't think we need to do anything atm. openchat provides different templated for coding mode https://huggingface.co/openchat/openchat_3.5/blob/main/README.md?code=true#L103-L105

Reply:
Yes we can ignore for now.